sdk/examples/how_tos/simple_transaction/simple_transaction.rs

```diff
@@ -33,15 +33,14 @@ async fn main() -> Result<()> {
             1_000_000,
         )];
         let transaction = account.send_amount(outputs, None).await?;
-        println!("Transaction sent: {}", transaction.transaction_id);
 
         // Wait for transaction to get included
         let block_id = account
             .retry_transaction_until_included(&transaction.transaction_id, None, None)
             .await?;
 
         println!(
-            "Block included: {}/block/{}",
+            "Block sent: {}/block/{}",
```

Comment:
It's a detail but @kwek20 harmonised all the examples like that, not sure we should start changing them again

Reply:
But I see that it's to harmonise with the other languages so whatever